Mode_selection.py

```diff
@@ -0,0 +1,11 @@
+import Pair_num_color as Pairnum_color
+from Colour_classes import MAJOR_COLORS,MINOR_COLORS
+class Mode:
+    def color_pair_to_string(self,major_color, minor_color):
+        return f'{major_color} {minor_color}'
+    def verfication(self):
```

Comment:
Method naming can be improved to make it meaningful.
This method is actually printing the manual, hence name which gives an indication of printing manual may be chosen.